### PR TITLE
[AMSDK-6963][AMSDK-6962] Mobile Core updates for collectLaunchInfo public API

### PR DESCRIPTION
**Pull request by emdobrin**

### CODE REVIEW

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md [hunk 1]
```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
 {% endtab %}
 {% endtabs %}
 
+
+
+## Collect launch information
+
+You can provide the user info to the SDK from various launch points in your application.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md]: please spell out "info"

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md [hunk 2]
```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
...
 
+
+
+## Collect launch information
+
+You can provide the user info to the SDK from various launch points in your application. 
+
+{% hint style="info" %}
+
+If **Analytics** extension is enabled in your SDK, the collection of this launch data will result in an Analytics request being sent. Other extensions in the SDK might use the collected data, for example, as a rule condition for an In-App Message.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md]: Please add "the" between "If" and "Analytics".

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md [hunk 3]
```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
...
+{% tab title="Android" %}
+
+#### Java
+
+In Android, you can collect data from the Activity or the context to be used later by the SDK.
+
+The `collectLaunchInfo` API marshals the Activity instance and extracts the intent data or extras. It should be called to support the following use cases:
+
+- Tracking Deep Link click-throughs
+  - Update `AndroidManifest.xml` to support intent-filter in the activity with the intended action and type of data.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md]: please add "the" between "support" and "intent" and remove the hyphen between "intent" and filter"

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md [hunk 4]
```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
...
+  - Handle the intent in the activity.
+  - Pass the Activity with deepLink intent to SDK in `collectLaunchInfo`
+- Tracking Push Message click-through
+  - Push message data must be added to the Intent used to open target activity on click-through.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md]: I'm setting "intent" with a lower case "i" and here with an upper case "I". Please make it consistent.

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md [hunk 5]
```diff
@@ -136,6 +136,115 @@ ACPCore.trackState("state name", data: ["key": "value"])
...
+  - The data can be added in intent extras which is then collected by SDK when the target activity is passed in `collectLaunchInfo`.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-api-reference.md]: please add a comman between "extras" and "which"

File: using-mobile-extensions/mobile-core/configuration-reference/mobile-core-event-reference.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+| :------------------------------- | :----------------------- | :----- | :-------- |
+| com.adobe.eventType.generic.data | com.adobe.eventSource.os | No     | N/A       |
+
+#### Data Payload Definition
+
+Here are the key-value pairs in this event:
+
+| **Key**        | **Value Type** | **Optional** | **Description**                                              |
+| :------------- | :------------- | :----------- | :----------------------------------------------------------- |
+| deeplink       | String         | Yes          | Deep link information collected from the provided user info  |
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/configuration-reference/mobile-core-event-reference.md]: please add periods at the end of the sentences in the description.

**Comment by emdobrin** [comment]
I will close this PR and open one for each platform since we are releasing iOS only for now

**Pull request closed without merging**